### ISSUE TITLE
MisconfigurationException: ModelCheckpoint(monitor='accuracy') not found in the returned metrics

### ISSUE THREAD

**Issue by 649459021** [shercoo/RGDiffSR#2]
Hi,

I put the pretrained VQGAN model in `checkpoints`, and then run the training script:
`python main.py -b configs/latent-diffusion/sr_best.yaml -t --gpus 0`

I encountered this error:
```
MisconfigurationException: ModelCheckpoint(monitor='accuracy') not found in the returned metrics:
['train/loss_simple', 'train/loss_simple_step', 'train/loss_vlb', 'train/loss_vlb_step', 'train/loss',
'train/loss_step', 'global_step', 'val/loss_simple', 'val/loss_vlb', 'val/loss',
'val/loss_simple_ema', 'val/loss_vlb_ema', 'val/loss_ema', 'train/loss_simple_epoch',
'train/loss_vlb_epoch', 'train/loss_epoch']. HINT: Did you call self.log('accuracy', value) in the
LightningModule?

```

**Comment by shercoo** [comment]
Please refer to `test.py`, `RecognizeCallback` . We used the recognition accuracy as monitor metric during training.
You can change the monitor keyword to val/loss, or duplicate the codes in `test.py` to do training.
We will update the codes later.

**Comment by shercoo** [comment]
main.py is updated.
